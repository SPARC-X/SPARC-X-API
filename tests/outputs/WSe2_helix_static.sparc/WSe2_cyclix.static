***************************************************************************
                            Atom positions                                 
***************************************************************************
Fractional coordinates of W:
      1.1972993167       0.0000145889       0.9999658380
Fractional coordinates of Se:
      1.0915974792       0.3392894457       0.0000104937
      1.3005058466       0.3291420709       0.0000236683
Total free energy (Ha): -3.055065628300148E+02
Atomic forces (Ha/Bohr):
 -1.9631231365E-04   2.0197957296E-03  -9.3342235956E-04
 -1.4864255791E-03  -8.3883915724E-04   5.1817786556E-04
 -1.5069571202E-02  -2.7428580192E-03   4.1524449400E-04
Stress (Ha/Bohr): 
  5.9636521405E-01 
Stress equiv. to all periodic (GPa): 
  2.5346951054E+00 
