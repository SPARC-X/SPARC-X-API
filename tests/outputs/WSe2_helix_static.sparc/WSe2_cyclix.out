***************************************************************************
*                       SPARC (version Aug 01, 2023)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Tue Sep  5 12:03:33 2023                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
CELL: 30.3498965822 0.299199300341885 3.1359066724 
TWIST_ANGLE: 0.047705 
FD_GRID: 120 42 25
FD_ORDER: 12
BC: D C H
KPOINT_GRID: 1 3 4
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.001
EXCHANGE_CORRELATION: GGA_PBE
NSTATES: 32
CHEB_DEGREE: 100
CHEFSI_BOUND_FLAG: 0
CALC_STRESS: 1
MAXIT_SCF: 100
MINIT_SCF: 2
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-06
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-08
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-09
MIXING_VARIABLE: potential
MIXING_PRECOND: none
MIXING_PARAMETER: 0.3
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
NUM_CHEFSI: 1
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 1
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_ENERGY_DENSITY: 0
OUTPUT_FILE: WSe2_cyclix/temp_run/WSe2_cyclix
***************************************************************************
                                Cell                                       
***************************************************************************
Volume :  1.0336846642E+03 (Bohr^3)
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 7
NP_BAND_PARAL: 6
NP_DOMAIN_PARAL: 1 1 1
NP_DOMAIN_PHI_PARAL: 8 3 2
EIG_SERIAL_MAXNS: 1500
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  48
Mesh spacing in x-direction        :  0.252916 (Bohr)
Mesh spacing in y-direction        :  0.00712379 (Bohr)
Mesh spacing in z-direction        :  0.125436 (Bohr)
Output printed to                  :  WSe2_cyclix/temp_run/WSe2_cyclix.out
Total number of atom types         :  2
Total number of atoms              :  3
Total number of electrons          :  46
Atom type 1  (valence electrons)   :  W 14
Pseudopotential                    :  ../psps/74_W_14_2.3_2.5_pbe_n_v1.0.psp8
Atomic mass                        :  183.84
Pseudocharge radii of atom type 1  :  7.33 0.52 7.15 (x, y, z dir)
Number of atoms of type 1          :  1
Atom type 2  (valence electrons)   :  Se 16
Pseudopotential                    :  ../psps/34_Se_16_1.9_2.1_pbe_n_v1.0.psp8
Atomic mass                        :  78.971
Pseudocharge radii of atom type 2  :  7.33 0.53 7.15 (x, y, z dir)
Number of atoms of type 2          :  2
Estimated total memory usage       :  3.06 GB
Estimated memory per processor     :  65.31 MB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -1.0183254549E+02        7.613E-02        27.229
2            -1.0183429867E+02        5.101E-02        7.144
3            -1.0183502413E+02        3.712E-02        7.149
4            -1.0183546552E+02        1.235E-02        7.108
5            -1.0183544523E+02        1.337E-02        9.091
6            -1.0183551633E+02        2.416E-03        10.378
7            -1.0183552075E+02        3.882E-04        7.066
8            -1.0183552093E+02        8.927E-05        6.673
9            -1.0183552093E+02        2.839E-05        6.699
10           -1.0183552094E+02        8.981E-06        7.334
11           -1.0183552092E+02        7.418E-06        7.366
12           -1.0183552094E+02        6.178E-07        7.352
Total number of SCF: 12    
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -1.0183552094E+02 (Ha/atom)
Total free energy                  : -3.0550656283E+02 (Ha)
Band structure energy              : -6.1063578525E+01 (Ha)
Exchange correlation energy        : -5.8895903012E+01 (Ha)
Self and correction energy         : -2.9306005079E+02 (Ha)
-Entropy*kb*T                      : -1.0332736669E-12 (Ha)
Fermi level                        : -1.7486456952E-01 (Ha)
RMS force                          :  6.4467289927E-03 (Ha/Bohr)
Maximum force                      :  1.5322782852E-02 (Ha/Bohr)
Time for force calculation         :  0.415 (sec)
Maximum stress                     :  5.9636521405E-01 (Ha/Bohr**2)
Maximum stress equiv. to periodic  :  5.7811301847E+02 (GPa)
Time for stress calculation        :  0.443 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  115.440 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*               List of contributors: See the documentation               *
*         Citation: See README.md or the documentation for details        *
*  Acknowledgements: U.S. DOE SC (DE-SC0019410), U.S. DOE NNSA (ASC)      *
*      {Preliminary developments: U.S. NSF (1333500,1663244,1553212)}     *
***************************************************************************
                                                                           
